packages/flutter/lib/src/rendering/object.dart

```diff
@@ -671,6 +672,105 @@ class _ForkingSemanticsFragment extends _SemanticsFragment {
   }
 }
 
+class RenderObjectOwner {
```

Comment:
I like your idea of calling this PipelineManager or some such.

It definitely shouldn't start with "Render".